### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by BelaM80**

### CODE REVIEW

File: client/src/components/Modal/Modal.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    border: none;
+    background-color: #fff;
+    width: 20px;
+    height: 20px;
+    cursor: pointer;
+    text-align: center;
+  }
+`;
+
+const Modal = ({ modalOpen, setModalOpen }) => {
```
Comment (boldCM) [inline client/src/components/Modal/Modal.js:28]: This is the only thing I'm not sure about...giving the useState through that component...but besides that, love it!:)

**Review by boldCM — APPROVED** [review]
lgtm 🎉

File: client/src/components/Modal/Modal.js (hunk 1 above)
Comment (BelaM80) [reply client/src/components/Modal/Modal.js:28]: Thanks for review.. Maybe in a later step, I might outsource the state into am custom hook.

File: client/src/components/Modal/Modal.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  const handleSubmit = (e) => {
+    e.preventDefault();
+  };
+
+  return (
+    <>
+      {modalOpen ? (
+        <ModalContainer modalOpen={modalOpen}>
+          <ModalHeader>
+            {" "}
```
Comment (phlgr) [inline client/src/components/Modal/Modal.js:51]: Remove this, as it is an anomaly from prettier

File: client/src/components/Modal/Modal.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  }
+`;
+
+const Modal = ({ modalOpen, setModalOpen }) => {
+  const handleSubmit = (e) => {
+    e.preventDefault();
+  };
+
+  return (
+    <>
```
Comment (phlgr) [inline client/src/components/Modal/Modal.js:47]: Are these fragments necessary

**Review by phlgr — CHANGES_REQUESTED** [review]

File: client/src/components/Modal/Modal.js (hunk 2 above)
Comment (BelaM80) [reply client/src/components/Modal/Modal.js:51]: Thanks, I didn´t even recognize it.. 😄

File: client/src/components/Modal/Modal.js (hunk 3 above)
Comment (BelaM80) [reply client/src/components/Modal/Modal.js:47]: Yes

**Pull request merged**